model_type: "MiDaS_small"  # MiDaS v2.1 - Small   (lowest accuracy, highest inference speed)
# model_type: "DPT_Large"     # MiDaS v3 - Large     (highest accuracy, slowest inference speed)
# model_type: "DPT_Hybrid"   # MiDaS v3 - Hybrid    (medium accuracy, medium inference speed)

device: "cpu"
# device: "cuda"
